src/plugins/core/pivot.ts

```diff
@@ -33,14 +46,18 @@ export class PivotCorePlugin extends CorePlugin<CoreState> implements CoreState
     [pivotId: UID]: Pivot | undefined;
   } = {};
   public readonly formulaIds: { [formulaId: UID]: UID | undefined } = {};
+  public readonly compiledMeasureFormulas: Record<UID, Record<string, RangeCompiledFormula>> = {};
```

Comment:
The issue is that it's currently not explicitly linked to a given pivot. And two pivots could have the same formula.
It's not explicit because where we use/create those computed measures, we only have the pivot definition and it doesn't contain the pivot id.

I could, upon pivot deletion, add something to check if other pivots depends on the same formula, and only remove the formula if there's none. or not clean.
what do you think?